actionview/test/template/render_test.rb

```diff
@@ -7,6 +7,14 @@ class TestController < ActionController::Base
 end
 
 module RenderTestCases
+  def self.included(test_case)
```

Comment:
How about just define `def teardown` and ensure calling `super` in the test classes which includes the module?